### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by noerw**
- fixes a minor bug in the gulp build chain, adds the command `npm run watch`
- extends the `addPanel()` function with the ability add links instead of panels (depends on #9)
- add more examples, structured to show separate features
- add autopan feature: maintains visibilty of the centered map point when opening the sidebar

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -33,13 +35,13 @@
...
             <ul role="tablist">
-                <li><a href="#home" role="tab"><i class="fa fa-bars"></i></a></li>
+                <li><a href="#home" role="tab"><i class="fa fa-bars active"></i></a></li>
                 <li><a href="#profile" role="tab"><i class="fa fa-user"></i></a></li>
                 <li class="disabled"><a href="#messages" role="tab"><i class="fa fa-envelope"></i></a></li>
             </ul>
 
             <ul role="tablist">
-                <li><a href="#settings" role="tab"><i class="fa fa-gear"></i></a></li>
+                <li><a href="//github.com/nickpeihl/leaflet-sidebar-v2"><i class="fa fa-github"></i></a></li>
```
Comment (nickpeihl) [inline examples/index.html]: I think protocol relative URLs is an anti-pattern. Can we just use https://? [More Info](https://www.paulirish.com/2010/the-protocol-relative-url/)

Comment (noerw) [reply examples/index.html]: :+1:

**Comment by nickpeihl** [comment]
@noerw You've put a lot of work into this and I appreciate it. I've added you as a maintainer for this package on npm. If you like you can `npm publish` your changes with respect to semver.

**Comment by noerw** [comment]
Thanks! I just released a couple of versions, #11 being a breaking change. Hope that's all right.
cheers!

**Pull request merged**